--- netactview-0.4.1/data/netactview.desktop.in.org	2009-08-21 19:26:33.000000000 +0200
+++ netactview-0.4.1/data/netactview.desktop.in	2009-08-21 19:26:53.000000000 +0200
@@ -1,11 +1,10 @@
 [Desktop Entry]
-Encoding=UTF-8
 _Name=Net Activity Viewer
 _Comment=View network connections
 TryExec=netactview
 Exec=netactview
-Icon=netactview.png
+Icon=netactview
 Terminal=false
 Type=Application
 StartupNotify=true
-Categories=GNOME;GTK;Application;Network;Monitor
+Categories=GNOME;GTK;Network;Monitor;
